modules/material/themes/material/core/loginuserpass.php

```diff
@@ -21,27 +21,28 @@ function preventDefault(event) {
     <?php
     $siteKey = htmlentities($this->data['recaptcha.siteKey']);
 
-    if (! empty($siteKey)) {
-    ?>
-    <script src='https://www.google.com/recaptcha/api.js?onload=onRecaptchaLoad&render=explicit' async defer></script>
-
-    <script>
-        function submitForm() {
-            document.querySelector('form').submit();
-        }
+    if (!empty($siteKey)) {
+        ?>
+        <script src='https://www.google.com/recaptcha/api.js?onload=onRecaptchaLoad&render=explicit' async
+                defer></script>
```

Comment:
I think the async probably should have moved with the defer for better wrapping here.